### PR TITLE
[#9657] Improve look of dropdown options in team contribution question 

### LINKED ISSUE #9657: Improve look of dropdown options in team contribution question
- **Environment**: `master` branch at commit 87a87e9e.

**Steps to reproduce**:
1. Go to the submission page of a feedback session that has a team contribution question. (eg. `/web/student/sessions/submission?courseid=test.exa-demo&fsname=Second%20team%20feedback%20session`)
2. Open the team contribution question's dropdown options

**Expected behaviour**
The colours of the options should be consistent with that of v6. `Equal share + ...` should be green, `Equal share - ...` should be red, and the remaining should be grey.

2019-04-10_11h41_422019-04-10_11h41_52

**Actual behaviour**
The colour of all the dropdown options is that of the option that was selected.

2019-04-10_11h41_142019-04-10_11h41_31

**Additional notes:**
While we're at it, the width of the dropdown in v7 can be reduced to look like the one in v6.

### PR DESCRIPTION
**Pull request by erjillsison**
Fixes #9657

Updated the html file to change the color of the contribution options based on point value.

How it looks after the change:
image

### CODE REVIEW

File: src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html [hunk 1]
```diff
@@ -6,8 +6,11 @@
               'color-neutral': responseDetails.answer === 100 || responseDetails.answer === CONTRIBUTION_POINT_NOT_SURE}"
             [ngModel]="responseDetails.answer" (ngModelChange)="triggerResponseDetailsChange('answer', $event)" [disabled]="isDisabled">
       <option [ngValue]="CONTRIBUTION_POINT_NOT_SUBMITTED"></option>
-      <option *ngFor="let point of contributionQuestionPoints" [ngValue]="point">{{ point | contributionPointDescription }}</option>
-      <option *ngIf="questionDetails.isNotSureAllowed" [ngValue]="CONTRIBUTION_POINT_NOT_SURE">Not Sure</option>
+      <option *ngFor="let point of contributionQuestionPoints" [ngValue]="point"
+              [ngClass]="{'color-positive': point > 100,
+              'color-negative': point < 100,
+              'color-neutral': point === 100}">{{ point | contributionPointDescription }}</option>
+      <option *ngIf="questionDetails.isNotSureAllowed" [ngValue]="CONTRIBUTION_POINT_NOT_SURE" [ngClass]="'color-neutral'">Not Sure</option>
```
Comment (xpdavid) [inline src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html]: If there is no need to use expression, you can just say
```suggestion
      <option *ngIf="questionDetails.isNotSureAllowed" [ngValue]="CONTRIBUTION_POINT_NOT_SURE" class="color-neutral">Not Sure</option>
```

File: src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html [hunk 2]
```diff
@@ -6,8 +6,11 @@
               'color-neutral': responseDetails.answer === 100 || responseDetails.answer === CONTRIBUTION_POINT_NOT_SURE}"
             [ngModel]="responseDetails.answer" (ngModelChange)="triggerResponseDetailsChange('answer', $event)" [disabled]="isDisabled">
       <option [ngValue]="CONTRIBUTION_POINT_NOT_SUBMITTED"></option>
-      <option *ngFor="let point of contributionQuestionPoints" [ngValue]="point">{{ point | contributionPointDescription }}</option>
-      <option *ngIf="questionDetails.isNotSureAllowed" [ngValue]="CONTRIBUTION_POINT_NOT_SURE">Not Sure</option>
+      <option *ngFor="let point of contributionQuestionPoints" [ngValue]="point"
+              [ngClass]="{'color-positive': point > 100,
+              'color-negative': point < 100,
```
Comment (xpdavid) [inline src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html]: The line has the wrong indentation?
```java
              [ngClass]="{'color-positive': point > 100,
                'color-negative': point < 100,
```

File: src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html (hunk 1 above)
Comment (erjillsison) [reply src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html]: Updated

File: src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html (hunk 2 above)
Comment (erjillsison) [reply src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html]: Updated

**Pull request merged**